routing/router.go

```diff
@@ -705,33 +706,26 @@ func (r *ChannelRouter) syncGraphWithChain() error {
 		// We're only interested in all prior outputs that have been
 		// spent in the block, so collate all the referenced previous
 		// outpoints within each tx and input.
-		var spentOutputs []*wire.OutPoint
 		for _, tx := range filterBlock.Transactions {
 			for _, txIn := range tx.TxIn {
 				spentOutputs = append(spentOutputs,
 					&txIn.PreviousOutPoint)
 			}
 		}
+	}
 
-		// With the spent outputs gathered, attempt to prune the
-		// channel graph, also passing in the hash+height of the block
-		// being pruned so the prune tip can be updated.
-		closedChans, err := r.cfg.Graph.PruneGraph(spentOutputs,
-			nextHash,
-			nextHeight)
-		if err != nil {
-			return err
-		}
-
-		numClosed := uint32(len(closedChans))
-		log.Infof("Block %v (height=%v) closed %v channels",
-			nextHash, nextHeight, numClosed)
-
-		numChansClosed += numClosed
+	// With the spent outputs gathered, attempt to prune the
+	// channel graph, also passing in the best hash+height
+	// so the prune tip can be updated.
+	closedChans, err := r.cfg.Graph.PruneGraph(spentOutputs,
+		bestHash,
```

Comment:
Style nit: in these cases, we prefer to write this as:

```go
r.cfg.Graph.PruneGraph(
    spentOutputs, bestHash, uint32(bestHeight),
)
```

Reply:
Fixed